From 19cd0e4728408a6de8ef4d6533369999d5f6af1a Mon Sep 17 00:00:00 2001
From: Evgeny Eltsin <eaeltsin@google.com>
Date: Fri, 25 Jun 2010 15:55:28 +0400
Subject: [PATCH 20/21] [gcc] Cleanup nacl_special_commands.

This seems like an artifact of an old refactoring.

Review URL: http://codereview.chromium.org/2824029
---
 gcc/gcc/emit-rtl.c |    6 +-----
 1 files changed, 1 insertions(+), 5 deletions(-)

diff --git a/gcc/gcc/emit-rtl.c b/gcc/gcc/emit-rtl.c
index 069137d..a91e7fd 100644
--- a/gcc/gcc/emit-rtl.c
+++ b/gcc/gcc/emit-rtl.c
@@ -852,10 +852,6 @@ byte_lowpart_offset (enum machine_mode outer_mode,
     return -subreg_lowpart_offset (inner_mode, outer_mode);
 }
 
-/* NaCl require temporary variables in places where usually it's
-   not allowed */
-int nacl_special_commands = 0;
-
 /* Generate a REG rtx for a new pseudo register of mode MODE.
    This pseudo is assigned the next sequential register number.  */
 
@@ -865,7 +861,7 @@ gen_reg_rtx (enum machine_mode mode)
   rtx val;
   unsigned int align = GET_MODE_ALIGNMENT (mode);
 
-  gcc_assert (can_create_pseudo_p () || nacl_special_commands);
+  gcc_assert (can_create_pseudo_p ());
 
   /* If a virtual register with bigger mode alignment is generated,
      increase stack alignment estimation because it might be spilled
-- 
1.7.1

